scripts/scripts.js

```diff
@@ -300,6 +300,45 @@ export function addTopSpacingStyleToFirstMatchingSection(main) {
   });
 }
 
+function getViewPort() {
+  const { width } = getWindowSize();
+  if (width >= 1232) {
+    return 'dekstop';
```

Comment:
Nit: Typo; Accidentally this still works since it's the default in the switch below